plugins/modules/configuration_item_batch.py

```diff
@@ -105,22 +105,29 @@ def update(module, table_client):
     cmdb_table = module.params["sys_class_name"]
     id_column_set = module.params["id_column_set"]
 
+    results = []
```

Comment:
Can you please update the `RETURN` block on line 95 to reflect this? Also, add a note in the documentation stating changes in 2.* and 1.* release.

Reply:
@juremedvesek Can you please address this?

Reply:
It should be fixed by commit:
https://github.com/ansible-collections/servicenow.itsm/pull/185/commits/79689d6b561c30c52082a57303c263deafa181e8